### DISCUSSION TITLE
How To Add YouTube Videos To Blog

### DISCUSSION THREAD

**Discussion by davesaah** [Ice-Hazymoon/hugo-theme-luna#57] [Q&A]
If there a way to add youtube videos with this theme?

**Comment by davesaah** [comment]
Dumb me 🤦🏾, I figured hugo has a built in shortcode for YouTube embeds.

**Comment by Ice-Hazymoon** [comment]
try: https://gohugo.io/content-management/shortcodes/#youtube

I haven't tried this shortcode, and I'm not sure if there are style issues

**Reply by davesaah** [reply]
Yeah, it actually works
